pydocmd.yml

```diff
@@ -60,7 +82,17 @@ pages:
       Products: endpoints/products.md
       Vessels: endpoints/vessels.md
       EIA Forecasts: endpoints/eia_forecasts.md
-      Ton Miles Breakdown: endpoints/ton_miles_breakdown.md
+      Tonne-miles Breakdown: endpoints/tonne_miles_breakdown.md
+      Vessel Availability: endpoints/availability_search.md
+      Vessel Availability Time Series: endpoints/availability_timeseries.md
```

Comment:
Can you please uniform all usage of "Time Series": either "Time Series" or "Timeseries" in all documents.